src/app/clients/ClientsPage.cy.tsx

```diff
@@ -0,0 +1,256 @@
+                    expect(clientTableData).to.deep.equal([
+                        {
+                            parcelId: "PRIMARY_KEY",
+                            flaggedForAttention: false,
+                            requiresFollowUpPhoneCall: true,
+                            fullName: "CLIENT_NAME",
+                            familyCategory: "Family of 3",
+                            addressPostcode: "SW1A 2AA",
+                            collectionCentre: "COLLECTION_CENTRE",
+                            congestionChargeApplies: true,
+                            packingDate: "04/08/2023",
+                            packingTimeLabel: "PM",
+                            lastStatus: "LAST_EVENT @ 04/08/2023",
+                        },
+                    ]);
+                })
+            );
+        });
+
+        it("familyCountToFamilyCategory()", () => {
+            expect(familyCountToFamilyCategory(1)).to.eq("Single");
+            expect(familyCountToFamilyCategory(2)).to.eq("Family of 2");
+            expect(familyCountToFamilyCategory(9)).to.eq("Family of 9");
+            expect(familyCountToFamilyCategory(10)).to.eq("Family of 10+");
+            expect(familyCountToFamilyCategory(15)).to.eq("Family of 10+");
+        });
+
+        it("formatDatetimeAsDate()", () => {
+            expect(formatDatetimeAsDate("2023-08-04T13:30:00+00:00")).to.eq("04/08/2023");
+            expect(formatDatetimeAsDate("2024-11-23T01:43:50+00:00")).to.eq("23/11/2024");
+            expect(formatDatetimeAsDate("Invalid_Date_Format")).to.eq("-");
+            expect(formatDatetimeAsDate(null)).to.eq("-");
+        });
+
+        it("datetimeToPackingTimeLabel()", () => {
+            expect(datetimeToPackingTimeLabel("2023-08-04T08:30:00+00:00")).to.eq("AM");
+            expect(datetimeToPackingTimeLabel("2023-08-04T13:30:00+00:00")).to.eq("PM");
+        });
+
+        it("eventToStatusMessage()", () => {
+            expect(
+                eventToStatusMessage({
+                    event_name: "EVENT",
+                    timestamp: "2023-08-04T13:30:00+00:00",
+                })
+            ).to.eq("EVENT @ 04/08/2023");
+            expect(eventToStatusMessage(null)).to.eq("-");
+        });
+    });
+
+    describe("Backend Processing for Expanded Client Details", () => {
+        it("Fields are set correctly", () => {
+            const expandedClientDetails = rawDataToExpandedClientDetails(
+                sampleRawExpandedClientDetails
+            );
+
+            expect(expandedClientDetails).to.deep.equal({
+                "voucher_#": "VOUCHER_1",
+                full_name: "CLIENT NAME",
+                phone_number: "PHONE NUMBER",
+                packing_date: "04/08/2023",
+                packing_time: new Date("2023-08-04T08:30:00+00:00").getTimezoneOffset()
+                    ? "14:30"
+                    : "13:30",
```

Comment:
dislike ternary since it only allows for +1 time zone support - can you do something like
```ts
const packingTimeInUTC = new Date("...")
const packingTimeInZone = new Date(d.getTime() + d.getTimezoneOffset() * 60000)}
```
and then converted to a string 'normally'?